python/hail/dataset.py

```diff
@@ -3459,50 +3474,39 @@ def pca(self, scores, loadings=None, eigenvalues=None, k=10, as_array=False):
 
         Separately, for the PCs PLINK/GCTA output the eigenvectors of the GRM; even ignoring the above discrepancy that means the left singular vectors :math:`U_k` instead of the component scores :math:`U_k S_k`. While this is just a matter of the scale on each PC, the scores have the advantage of representing true projections of the data onto features with the variance of a score reflecting the variance explained by the corresponding feature. (In PC bi-plots this amounts to a change in aspect ratio; for use of PCs as covariates in regression it is immaterial.)
 
-        **Annotations**
-
-        Given root ``scores='sa.scores'`` and ``as_array=False``, :py:meth:`~hail.VariantDataset.pca` adds a Struct to sample annotations:
 
-         - **sa.scores** (*Struct*) -- Struct of sample scores
+        Scores are stored in a KeyTable with the following structure:
 
-        With ``k=3``, the Struct has three field:
+        **s** (*Sample*) The sample ID.
 
-         - **sa.scores.PC1** (*Double*) -- Score from first PC
+        **pcaScores** The principal component scores. This can have two different structures, depending on the value of the ``as_array`` flag.
 
-         - **sa.scores.PC2** (*Double*) -- Score from second PC
+        If ``as_array`` is ``False`` (the default), then ``pcaScores`` is a Struct with fields ``PC1``, ``PC2``, etc. If ``as_array`` is ``True``, then ``pcaScores`` is a double-valued array containing the principal component scores.
 
-         - **sa.scores.PC3** (*Double*) -- Score from third PC
+        Loadings are stored in a KeyTable with a structure similar to the scores KeyTable:
 
-        Analogous variant and global annotations of type Struct are added by specifying the ``loadings`` and ``eigenvalues`` arguments, respectively.
+        **v** (*Variant*) The variant ID.
 
-        Given roots ``scores='sa.scores'``, ``loadings='va.loadings'``, and ``eigenvalues='global.evals'``, and ``as_array=True``, :py:meth:`~hail.VariantDataset.pca` adds the following annotations:
+        **pcaLoadings** (*Struct* or *Array*) The variant loadings.
 
-         - **sa.scores** (*Array[Double]*) -- Array of sample scores from the top k PCs
+        Eigenvalues are returned as an array of doubles.
```

Comment:
You might mention this is a Pyton array of doubles.

Reply:
Nor not.  Return type in the docs also make this totally clear.

Reply:
I guess I should also use "list", since that's what python arrays are called.